### DISCUSSION TITLE
Use Markor (Android) together with Obsidian (PC)

### DISCUSSION THREAD

**Discussion by mslagmu** [gsantner/markor#1347] [Show and tell]
Hello

I'm user of Obsidian on my PC and with syncthing i synchronize my pc with my smar.phone. Markor is almost perfect for my using except one thing. The internal links. You use the symbol @ and obisdian use [[ toOtherNote ]]

you will find all information hier https://obsidian.md/features

Is it possible to put in your program a configuration field to define how the internal links must to be written ( for example with a regexp ) or at least to tell me in which file i need to see to make the changes myself?

Thank you

**Comment by adelobosko** [comment]
I am using the obsidian as well, and links to notes are working for me

https://user-images.githubusercontent.com/56083544/118771050-b82ebb00-b88a-11eb-91ec-dcd39eb9bb5b.mp4
